homeassistant/components/mqtt/strings.json

```
@@ -50,6 +53,7 @@
       "bad_client_cert": "Invalid client certiticate, ensure a PEM coded file is supplied",
       "bad_client_key": "Invalid private key, ensure a PEM coded file is supplied without password",
       "bad_client_cert_key": "Client certificate and private are no valid pair",
+      "bad_ws_headers": "Supply a valid JSON payload",
```

Maybe something like
```suggestion
      "bad_ws_headers": "Supply valid HTTP headers as a JSON object",
```
would be more user-friendly? "A valid JSON payload" could be anything...